site/content/docs/v1.11/customize-installation.md

```diff
@@ -135,7 +135,7 @@ kubectl patch deployment velero -n velero --patch \
 
 **node-agent pod**
 
-Update the `spec.template.spec.containers.resources.limits` and `spec.template.spec.containers.resources.requests` values in the node-agent DaemonSet spec.
+Update the `spec.template.spec.containers.resources.limits` and `spec.template.spec.containers.resources.requests` values in the node-agent DeamonSet spec.
```

Comment:
Same here. `DaemonSet`.

Reply:
These 3 files I reverted to what is on the main branch because I understood we should only change the `site/content/docs/main/` folder

Reply:
Got it.